ansible_collections/arista/avd/molecule/eos_cli_config_gen_deprecated_vars/documentation/devices/host2.md

```diff
@@ -2,11 +2,51 @@
 
 ## Table of Contents
 
+- [Authentication](#authentication)
+  - [AAA Accounting](#aaa-accounting)
 - [Interfaces](#interfaces)
   - [VXLAN Interface](#vxlan-interface)
 - [Routing](#routing)
   - [Router BGP](#router-bgp)
 
+## Authentication
+
+### AAA Accounting
+
+#### AAA Accounting Summary
+
+| Type | Commands | Record type | Group | Logging |
+| ---- | -------- | ----------- | ----- | ------- |
+| Exec - Console | - | start-stop | TACACS | True |
+| Commands - Console | all | start-stop | TACACS | True |
+| Commands - Console | 0 | start-stop |  -  | True |
+| Commands - Console | 1 | start-stop | TACACS1 | False |
+| Commands - Console | 2 | none | - | - |
+| Commands - Console | 3 | start-stop |  -  | False |
```

Comment:
I still don't get why we render in documentation, entries which are not in CLI. it should not be the case right?

Reply:
ok this is legacy syntax - we can either fix it or given that it is deprecated forget about it.